dist/blog.html

```diff
@@ -0,0 +1,389 @@
+<!doctype html>
```

Comment:
Left this file named as dist/index.html, since it's the main one. It will be easier to access it.